demos/cypress-shared/cypress/e2e/correct.captcha.cy.ts

```diff
@@ -30,7 +30,6 @@ describe('Captchas', () => {
                 context: { datasetWithSolutionHashes },
             })
         }
-        cy.intercept('/dummy').as('dummy')
```

Comment:
why has this been removed?

Reply:
I think it was just copied from stackoverflow by mistake. We don't have any API calls with `dummy` in them!